content/applications/websites/livechat.rst

```diff
@@ -211,26 +240,26 @@ it will open a new chat.
 Participate in a conversation
 =============================
 
-As explained above, *operators* are the users who will respond to live chat requests from customers.
-The information below outlines the necessary steps for operators participating in live chat
+As explained above, *operators* are the users who respond to live chat requests from customers. The
+information below outlines the necessary steps for operators participating in live chat
 conversations on an Odoo database.
 
 Set an online chat name
 -----------------------
 
-Before participating in a live chat, operators should update their *Online Chat Name*. This is the
-name that will be displayed to site visitors in the live chat conversation.
+Before participating in a live chat, operators may choose to update their *Online Chat Name*. This
+is the name that is displayed to site visitors in the live chat conversation.
 
-To update the :guilabel:`Online Chat Name`, click on the user name in the upper-right corner of any
-page in the database. Select :guilabel:`My Profile` to open the Profile page. On the right side of
-the :guilabel:`Preferences` tab, locate the :guilabel:`Online Chat Name` field and enter the
-preferred name.
+To update the :guilabel:`Online Chat Name`, click on the user avatar in the top-right corner of any
```

Comment:
Since the 'Online Chat Name' field still, at this point in the instructions, is NOT present on the UI, perhaps this would be an opportunity to use ** instead of GUI. Does that make sense? What are your thoughts on that?